src/applications/personalization/profile-2/msw-mocks.js

```diff
@@ -0,0 +1,62 @@
+import { rest } from 'msw';
+
+export const newPaymentAccount = {
+  accountType: 'Savings',
+  financialInstitutionName: 'COMERICA BANK',
+  accountNumber: '*****6789',
+  financialInstitutionRoutingNumber: '*****4321',
+};
+
+export const updateDirectDepositSuccess = [
+  rest.put(
+    // I'd prefer to just set the route as `ppiu/payment_information` or at least `v0/ppiu/payment_information`
+    'https://dev-api.va.gov/v0/ppiu/payment_information',
```

Comment:
This PR is failing in CI and my hunch is it's due to endpoint path not being valid in the CI environment.